src/lib/select/select.ts

```diff
@@ -175,6 +177,27 @@ export const _MatSelectMixinBase:
         mixinDisableRipple(mixinTabIndex(mixinDisabled(mixinErrorState(MatSelectBase))));
 
 
+/** Possibe rendering strategies for the MatSelect panel. */
+export enum MatSelectRenderingStrategy {
+  /**
+   * With the default rendering strategy, the content of a `MatSelect` will only be rendered
+   * in the DOM while the select is open. This strategy has better performance, however its
+   * accessibility is worse.
+   */
+  DEFAULT,
```

Comment:
I'd rather make these values something like `ONLY_WHEN_OPEN` and `ALWAYS`. We might someday change which one is the default, and "inline" makes me think that the options will be rendered inside of the `<mat-select>` element.

Alternatively, we could just make it a boolean like `optionsAlwaysInDom`